vendor/vendor.json

```diff
@@ -44,6 +38,18 @@
 			"revision": "d2f86524cced5186554df90d92529757d22c1cb6",
 			"revisionTime": "2016-08-23T09:51:56Z"
 		},
+		{
+			"checksumSHA1": "tMUbiAxGypX/0kCJpIFg31Q6vOU=",
+			"path": "github.com/marten-seemann/qtls",
```

Comment:
This needs a license. Since it's forked from tris, does tris have one?

Reply:
It doesn't. I opened an issue there: https://github.com/cloudflare/tls-tris/issues/147.